src/mmw/apps/geoprocessing_api/tasks.py

```diff
@@ -229,3 +230,112 @@ def collect_climate(results):
             'categories': categories
         }
     }
+
+
+@shared_task
+def analyze_terrain(result):
+    """
+    Starts a job to produce summary statistics for elevation and slope in a
+    given area.
+
+    TODO Where does the NED and Slope data come from?
+
+    For more information, see the
+    [technical documentation](https://wikiwatershed.org/
+    documentation/mmw-tech/#overlays-tab-coverage).
+
+    ## Response
+
+    You can use the URL provided in the response's `Location`
+    header to poll for the job's results.
+
+    <summary>
+       **Example of a completed job's `result`**
+    </summary>
+
+    <details>
+
+        {
+            "survey": {
+                "displayName": "Soil",
+                "name": "soil",
+                "categories": [
+                    {
+                        "elevation": 25.03116786250801,
+                        "slope": 2.708598957407307,
+                        "type": "average"
+                    },
+                    {
+                        "elevation": -0.84,
+                        "slope": 0.0,
+                        "type": "minimum"
+                    },
+                    {
+                        "elevation": 105.01,
+                        "slope": 44.52286911010742,
+                        "type": "maximum"
+                    }
+                ]
+            }
+        }
+
+    </details>
+
+    ---
+    type:
+      job:
+        required: true
+        type: string
+      status:
+        required: true
+        type: string
+
+    omit_serializer: true
+    parameters:
+       - name: body
+         description: A valid single-ringed Multipolygon GeoJSON
+                      representation of the shape to analyze.
+                      See the GeoJSON spec
+                      https://tools.ietf.org/html/rfc7946#section-3.1.7
+         paramType: body
+         type: object
+       - name: wkaoi
+         description: The table and ID for a well-known area of interest,
+                      such as a HUC.
+                      Format "table__id", eg. "huc12__55174" will analyze
+                      the HUC-12 City of Philadelphia-Schuylkill River.
+         type: string
+         paramType: query
+
+       - name: Authorization
+         paramType: header
+         description: Format "Token&nbsp;YOUR_API_TOKEN_HERE". When using
+                      Swagger you may wish to set this for all requests via
+                      the field at the top right of the page.
+    consumes:
+        - application/json
+    produces:
+        - application/json
+    """
+    if 'error' in result:
+        raise Exception('[analyze_terrain] {}'.format(result['error']))
+
+    categories = [
+        dict(type='average',
+             elevation=(result[0]['avg'] * M_PER_CM),
+             slope=result[1]['avg']),
```

Comment:
This is probably fine since we're just unpacking the results but if we wanted to assign these beforehand I think we could do:

```python
[elevation, slope] = result
categories = [
...
```

and then use `elevation` and `slope` in place of `result[0]` & `result[1]`. Arguable whether that's an improvement, though.

Reply:
I like it! Will fix.